src/entities/index.ts

```diff
@@ -1,7 +1,5 @@
 import { User } from './user.entity';
 
-export {
-    User,
-}
+export { User };
```

Comment:
🚫 **[eslint]** <[unicorn/prefer-export-from](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v56.0.0/docs/rules/prefer-export-from.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use `export…from` to re-export `User`.
<details><summary>reviewdog suggestion error</summary>GitHub comment range and suggestion line range must be same. L3-L3 v.s. L3-L6</details>

<!-- __reviewdog__:ChA0MTIzMzFjNTVkYzgxZWQ5EgZlc2xpbnQ= -->